kurtosis-devnet/simple.yaml

```diff
@@ -61,11 +61,18 @@ optimism_package:
     l2_artifacts_locator: {{ localContractArtifacts "l2" }}
     global_deploy_overrides:
       faultGameAbsolutePrestate: {{ localPrestate.Hashes.prestate }}
+      useSoulGasToken: true
+      soulGasTokenBlock: 0
+      isSoulBackedByNative: true
+      l2GenesisBlobTimeOffset: "0x0"
   global_log_level: "info"
   global_node_selectors: {}
   global_tolerations: []
   persistent: false
 ethereum_package:
+  participants:
+    - el_type: geth
+      cl_type: teku
```

Comment:
This is ported from [here](https://github.com/ethereum-optimism/optimism/pull/14305) .